spec/models/member_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Member, type: :model do
+  context "without a name" do
+    it "should be invalid" do
+      member = Member.new
+
+      member.email = 'gala@example.com'
+
+      expect(member).to be_invalid
+    end
+  end
+
+  context "without an email" do
+    it "should be invalid" do
+      member = Member.new
+
+      member.name = 'Zaba'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.